README.md

```diff
@@ -12,3 +27,33 @@ AdaptiveTheme sub-themes also require an additional module to work correctly. Pl
 Once you have downloaded and extracted the AdaptiveTheme framework and enabled the associated module, create a /custom folder inside your /themes folder if it doesn't already exist (not necessary but highly recommended). Move to the newly created custom folder and clone this repository inside. Alternatively, you can also download the compressed repository and extract in the custom folder. Once you have the files in the appropriate directory you should see the Carapace for Islandora theme in the appearance tab. Click the 'Install and set as default' link and you should be ready to go.
```

Comment:
Remove `and enabled the associated module` as this is a reference to `at_tools`

Reply:
removed at_tools references